scalafmt-core/shared/src/main/scala/org/scalafmt/util/MarkdownFile.scala

```diff
@@ -0,0 +1,133 @@
+package org.scalafmt.util
+
+import scala.meta.inputs.Position
+import scala.meta.inputs.Input
+import scala.collection.mutable
+
+final case class MarkdownFile(input: Input, parts: List[MarkdownPart]) {
+  private val appends = mutable.ListBuffer.empty[String]
+  def appendText(text: String): Unit = {
```

Comment:
this method is not used in this patch, nor is `appends`. remove?

Reply:
Sounds good. I just assumed Olafurpg had done it for some reason I didn't understand :)
Removed